corehq/apps/data_analytics/malt_generator.py

```diff
@@ -97,7 +98,6 @@ def _get_forms_queryset(self, user_id, domain_name, monthspan):
 
         return FormData.objects.exclude(
             device_id=COMMCONNECT_DEVICE_ID,
-            app_id=MISSING_APP_ID
```

Comment:
You might wanna take a look at [`Q()` objects](https://docs.djangoproject.com/en/1.8/topics/db/queries/#complex-lookups-with-q).  They let you encapsulate bits of queryest to combine using `&` (AND) and `|` (OR).